internal/pkg/describe/backend_service.go

```diff
@@ -15,45 +15,45 @@ import (
 	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/term/color"
 )
 
-// BackendAppDescriber retrieves information about a backend application.
-type BackendAppDescriber struct {
-	app             *config.Service
+// BackendServiceDescriber retrieves information about a backend service.
+type BackendServiceDescriber struct {
+	service         *config.Service
 	enableResources bool
 
-	store            storeSvc
-	appDescriber     appDescriber
-	initAppDescriber func(string) error
+	store                storeSvc
+	svcDescriber         svcDescriber
+	initServiceDescriber func(string) error
 }
 
-// NewBackendAppDescriber instantiates a backend application describer.
-func NewBackendAppDescriber(project, app string) (*BackendAppDescriber, error) {
-	svc, err := config.NewStore()
+// NewBackendServiceDescriber instantiates a backend service describer.
+func NewBackendServiceDescriber(app, svc string) (*BackendServiceDescriber, error) {
+	store, err := config.NewStore()
 	if err != nil {
 		return nil, fmt.Errorf("connect to store: %w", err)
 	}
-	meta, err := svc.GetService(project, app)
+	meta, err := store.GetService(app, svc)
 	if err != nil {
 		return nil, err
 	}
-	opts := &BackendAppDescriber{
-		app:             meta,
+	opts := &BackendServiceDescriber{
+		service:         meta,
 		enableResources: false,
-		store:           svc,
+		store:           store,
 	}
-	opts.initAppDescriber = func(env string) error {
-		d, err := NewAppDescriber(project, env, app)
+	opts.initServiceDescriber = func(env string) error {
+		d, err := NewServiceDescriber(app, env, svc)
 		if err != nil {
 			return err
 		}
-		opts.appDescriber = d
+		opts.svcDescriber = d
 		return nil
 	}
 	return opts, nil
 }
 
-// NewBackendAppDescriberWithResources instantiates a backend application with stack resources.
-func NewBackendAppDescriberWithResources(project, app string) (*BackendAppDescriber, error) {
-	d, err := NewBackendAppDescriber(project, app)
+// NewBackendServiceDescriberWithResources instantiates a backend service with stack resources.
```

Comment:
nit: i know that it was missing before :P but it just disambiguates  the constructor
```suggestion
// NewBackendServiceDescriberWithResources instantiates a backend service describer with stack resources.
```